clang/test/CodeGen/systemz-charset.c

```diff
@@ -0,0 +1,35 @@
+// RUN: %clang_cc1 %s -emit-llvm -triple s390x-none-zos -fexec-charset IBM-1047 -o - | FileCheck %s
+// RUN: %clang %s -emit-llvm -S -target s390x-ibm-zos -o - | FileCheck %s
```

Comment:
There should be tests for other converters.
`__STDC_MB_MIGHT_NEQ_WC__` should be defined in some cases.